cypress/fixtures/profile.json

```diff
@@ -0,0 +1,5 @@
+{
+  "id": 8739,
+  "name": "Jane",
+  "email": "jane@example.com"
+}
```

Comment:
EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon:
![image](https://user-images.githubusercontent.com/10209180/126383914-09439b8d-be81-43f1-ba59-df0ccd62f0f6.png)